src/store/listing/getters.js

```diff
@@ -1,7 +1,8 @@
-const getHrefFromId = (state) => (id) => {
-  return state.href.get(id)
-}
+const getHrefFromId = (state) => (id) => { return state.href.get(id) }
```

Comment:
You don't need this sintax if you return in same line. 
You can as well as this exemple: 

`const getHrefFromId = (state) => (id) => state.href.get(id)`

What do you think ?

Reply:
I agree, done!